_posts/2021-07-19-dd_00002_transistor.md

```diff
@@ -0,0 +1,236 @@
+</figure>
+
+You might have noticed the same three terminals (gate, source, and drain) we
+introduced earlier, this time with their common abbreviations (G, S, D), as they
+are used by engineers. "B" stands for **bulk** (sometimes also called _body_ or
+_substrate_). The bulk is basically the basis on top of which the transistor is
+built (if you remember from the previous post this basis is nothing but the
+actual _wafer_). Bulk, source, and drain are made from **silicon**. The gate on
+the top can theoretically be made of any metal, usually, poly-silicon is used
+today. Between the gate "plate" and the rest of the transistor, we can see
+another layer called **oxide**. The role of this oxide layer is to be an
+_insulating_ layer so that no current can flow between the gate and the bulk (or
+source/drain).
+
+When we apply a voltage at the gate terminal, a conducting **channel** starts to
+form between the source and the drain (see picture above). This channel then
+allows electrons to flow from source to drain. The voltage we need to apply to
+the gate such that the channel opens and a current can flow between source and
+drain, has a special name: **threshold voltage**. For any gate voltage below the
+threshold voltage, the transistor will not let any current pass from source to
+drain. The threshold voltage is commonly abbreviated as _Vth_.
+
+Let's see what goes on inside a transistor.
+
+When there is no voltage present at the gate, the channel between source and
+drain is not conductive, i.e., no current can flow from source to drain. The
+MOSFET is switched _off_. The following picture depicts this state (notice how
+the electrons are waiting at the source terminal):
+
+<figure class="img-center" style="width:50%">
+  <img src="/assets/images/dd_00002-mosfet-channel-off.png" alt="">
+  <!-- <figcaption class="figure-caption text-center">Microchips</figcaption> -->
+</figure>
+
+When we now start applying a _positive_ voltage at the gate terminal, the gate
+will attract **electrons** from the source terminal (remember that the source is
+made from Silicon which has some free electrons). As we increase the gate
+voltage, more and more electrons from the source will _accumulate_ near the gate
+(the area where the channel is supposed to be). Note that the electrons actually
+would love to get to the positively charged gate, but they can't because of the
+insulating oxide layer between gate and bulk. When the gate voltage reaches the
+_threshold voltage_, there are now enough electrons at the channel area to allow
+a current flow between source and drain. Imagine the accumulated electrons in
+the channel as a "bridge" between source and drain. Now electrons can flow from
+source to drain; the MOSFET is switched _on_:
+
+<figure class="img-center" style="width:50%">
+  <img src="/assets/images/dd_00002-mosfet-channel-on.png" alt="">
+  <!-- <figcaption class="figure-caption text-center">Microchips</figcaption> -->
+</figure>
+
```

Comment:
At this point it feels like a short break is necessary. First, I though you could link a video already here, but it can be confusing with NMOS/PMOS. Maybe you can say watch the first x seconds of the video. Or maybe just something like: "Sounds complicated? Don't worry ...."

Reply:
Agreed. Added something similar to the "Don't worry" suggestion.